backend/Tasque.BLL/Services/TaskService.cs

```diff
@@ -38,9 +38,11 @@ public class TaskService : ITaskService
         public async Task<TaskDto> CreateTask(TaskDto model)
         {
             var entity = _mapper.Map<Common.Entities.Task>(model);
-            var project = _dbContext.Projects.FirstOrDefault(p => p.Id == model.ProjectId)?? throw new CustomNotFoundException("project");
+            var project = _dbContext.Projects.FirstOrDefault(p => p.Id == model.ProjectId) ?? throw new CustomNotFoundException("project");            
 
             entity.Key = project.Key + '-' + UpdateProjectCounter(project.Id);
+            var order = _dbContext.Tasks.Where(t => t.ProjectId == model.ProjectId).Max(x => x.Order) + 1;
+            entity.Order = order;
```

Comment:
Is it possible to change this order?

Reply:
After creation task is placed last, and we can move it later